creating-virtual-machines/interfaces-and-networks.adoc

```diff
@@ -419,6 +419,25 @@ an IP address configured, which may introduce issues with third-party solutions
 that may rely on it. For example, Istio may not work in this mode.
 ______________________________________________________________________________
 
+______________________________________________________________________________
```

Comment:
as an uninitiated reader, the very first question I'll have is: why would I want to avoid bridge mode? Is it because of the other note above about third-party solutions? No other reasons? (There are other reasons though, specifically, migration issues as well as issues with OpenShift that I, as a reader, would probably appreciate to know about to make the right decision.) So should we document these other reasons to make the new text more helpful?